docker/exporter/export_runner.py

```diff
@@ -74,6 +76,39 @@ def spawn_ecosystem_exporter(work_dir: str, bucket: str, eco: str):
     logging.error('Export of %s failed with Exit Code: %d', eco, return_code)
 
 
+def aggregate_all_vulnerabilities(work_dir: str, export_bucket: str):
+  """
+  Aggregates vulnerability records from each ecosystem into a single zip
+  file and uploads it to the export bucket.
+  """
+  zip_file_name = 'all.zip'
+  output_zip = os.path.join(work_dir, zip_file_name)
+  all_vulns = {}
+
+  for ecosystem_dir, _, files in os.walk(work_dir):
+    if ecosystem_dir == work_dir:
+      continue
+    for vuln_filename in files:
+      if not vuln_filename.endswith('.json'):
+        continue
+      all_vulns[vuln_filename] = os.path.join(ecosystem_dir, vuln_filename)
+
+  with z.ZipFile(output_zip, 'a') as all_zip:
+    for vuln_filename in sorted(all_vulns):
```

Comment:
Curious if there's any particular reason to sort these?

Reply:
I remember we encountered some all.zip issue [because it was not sorted](https://github.com/google/osv.dev/pull/2311). So I think it would be better to sort this all.zip as well. (But I guess it's not necessary, as I got all the file names through the disk directory, which should already be sorted. But I still feel better to sort it just in case some race conditions.)